src/views/dashboard/playground/docs/getting-started.md

```diff
@@ -172,51 +114,151 @@ Multiple fields can be grouped together. The following SQL statement returns the
 SELECT hostname, environment,
     mean(usage_user) as usage_user_avg
     FROM cpu_metrics
-    GROUP BY hostname, environment;
+    GROUP BY hostname, environment
+    LIMIT 100;
 ```
 
-95 percent usage reflects the peak usage of the CPU. The following SQL statement returns the 95 percent CPU usage group by `hostname` and `environment`.
+95 percent usage reflects the peak usage of the CPU.
```

Comment:
In most case we use p95 for latency/response time. And it's rarely used to measure CPU usage. We need a better example and we can import another table with latency data if needed.